.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
-  - "8.0"
-before_script: "npm run validate"
```

Comment:
See my comment about having a discrete validation step. Leave this in the travis config